operations/exports.md

```diff
@@ -0,0 +1,158 @@
+        "706dc6d5-9511-4751-825e-538ce99da2ce",
+        "f776f20a-6f1a-4ddf-93f4-9dae95261415"
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `ExportIds` | array of string | optional, max 1000 items | Unique identifiers of the exports. If not specified, the operation returns exports with Status of `Pending`, `Processing`, and `Success` for the enterprise associated with the Access Token. |
+| `EntityType` | string [Entity type](#entity-type) | optional | Return only exports with the provided [Entity type](#entity-type). |
+| `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of message data returned (using cursor pagination). |
+
+### Response
+
+```json
+{
+    "Exports": [
+        {
+            "Id": "3fa85f64-5717-4562-b3fd-2c963f66afa6",
+            "Status": "Success",
+            "EntityType": "OrderItem",
+            "ExpiresUtc": "2023-10-26T11:42:28Z",
+            "Files": [
+                {
+                    "Url": "https://example.com/exports/3fa85f64-5717-4562-b3fd-2c963f66afa6-1.jsonl?example=signature",
+                    "SizeInBytes": 1215279
+                },
+                {
+                    "Url": "https://example.com/exports/3fa85f64-5717-4562-b3fd-2c963f66afa6-2.jsonl?example=signature",
+                    "SizeInBytes": 1398362
+                },
+            ]
+        },
+        {
+            "Id": "09708665-0e31-4b23-b337-b0a000be0df0",
+            "Status": "Pending",
+            "EntityType": "OrderItem",
+            "Files": [],
+            "ExpiresUtc": null
+        },
+        {
+            "Id": "706dc6d5-9511-4751-825e-538ce99da2ce",
+            "Status": "Processing",
+            "EntityType": "OrderItem",
+            "Files": [],
+            "ExpiresUtc": null
+        },
+        {
+            "Id": "706dc6d5-9511-4751-825e-538ce99da2ce",
+            "Status": "Expired",
+            "EntityType": "OrderItem",
+            "Files": [],
+            "ExpiresUtc": "2023-10-24T14:12:30Z"
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Exports` | array of [Exports](#export) | required | Requested exports. |
+
+#### Export
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the export. |
+| `Status` | string [Export status](#export-status) | required | Current status of the export. |
+| `EntityType` | string [Entity type](#entity-type) | required | Type of exported entities. |
+| `Files` | array of [Exported files](#exported-file) | required | Files with exported data, if available. |
```

Comment:
```suggestion
| `Files` | array of [Exported files](#exported-file) | required | Files with exported data. Empty if no files are available. |
```
I would change it little bit because at first reading I was like, why is it required when you are saying "if available" so I was thinking it can be null 😁 ?